auth/api/http/groups/transport.go

```diff
@@ -282,6 +282,19 @@ func decodeAssignRequest(_ context.Context, r *http.Request) (interface{}, error
 	return req, nil
 }
 
+func decodeUnassignRequest(_ context.Context, r *http.Request) (interface{}, error) {
+	req := unassignReq{
+		token:   r.Header.Get("Authorization"),
+		groupID: bone.GetValue(r, "groupID"),
+	}
+
+	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
```

Comment:
Can you simply send an array as a root element in request (like `["id1", "id2"...]`), so that you can later use
```go
if err := json.NewDecoder(r.Body).Decode(&req.members); err != nil {
```
There is no need for a root object with a `members` array inside.